workflow-init/profile/hawk.yml

```diff
@@ -236,3 +236,15 @@ component:
       - *backend_rest
       - *backend_grouping
       - *backend_search
+
+  configuration:
+    parameters:
+      <<: *defaultParameters
+      kustomize: bitbucket-fake
+      java-version: 17
```

Comment:
java-version is common parameter

you can add many free form-parameters in the future in case you need them, depending on worklfow